hooks/src/index.js

```diff
@@ -258,10 +258,11 @@ export function useErrorBoundary(cb) {
 function flushAfterPaintEffects() {
 	afterPaintEffects.some(component => {
 		if (component._parentDom) {
+			const effects = component.__hooks._pendingEffects;
+			component.__hooks._pendingEffects = [];
```

Comment:
can't we move this to `catch` and its original position?

Reply:
Do you mean putting `component.__hooks._pendingEffects = []` in _both_ the catch block and its original position, or something else?

Reply:
I mean that yes, will save bytes since the allocation will probably be the expensive part here

Reply:
Are you referring to minified size or gzipped size? If the latter I'm not a fan of gzip-golf but I'll defer to whatever the team normally prioritises.

Reply:
The community aims at minzip size so we are prioritizing that as well

Reply:
I mean even minified it's smaller

Reply:
OK, I've updated this accordingly.